assets/css/style.css

```diff
@@ -392,3 +392,104 @@ main {
 .hidden {
   display: none;
 }
+
+.contact-section {
+  width: 100%;
+  background-image: url('../images/backgrounds/contact-me.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+}
+
+.contact-background {
+  padding: 2em 2em 0 2em;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5em 0;
+  margin-bottom: 0;
+  background-position: right 15px;
+  background-repeat: no-repeat;
+  max-width: 30em;
+}
```

Comment:
- [ ] Please make sure your design is the same as the given Figma design. Right now, your contact form doesn't have a background image. Kindly, implement it.

![image](https://github.com/chudisoft/Portfolio-Microverse/assets/101523582/b2ca2ab2-3bd8-43de-b015-84f5d39adce2)
Your design
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/101523582/efe3acaf-dcc8-44df-a64f-e4df31d9a923)
 
Figma design
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/101523582/313e8858-cf30-42c6-844f-d2c02954f712)
 
- [ ] Please implement the top left border-radius of your contact form as the Figma design
- [ ] Please make sure you used the same back-ground colour to the Figma design